sovtoken/sovtoken/test/auth_map/test_auth_xfer.py

```diff
@@ -22,15 +27,15 @@ def mint_tokens(helpers, addresses):
 def do_transfer(txnPoolNodeSet, sdk_pool_handle, helpers, looper, sdk_wallet, addresses, amount, sign=False):
     [address_giver, address_receiver] = addresses
 
-    utxos = helpers.general.get_utxo_addresses([address_giver])[0]
-    inputs = [{ADDRESS: utxo["address"], SEQNO: utxo["seqNo"]} for utxo in utxos]
-    change = sum([utxo["amount"] for utxo in utxos]) - amount
+    inputs = helpers.general.get_utxo_addresses([address_giver])[0]
+    change = sum([utxo["amount"] for utxo in inputs]) - amount
     outputs = [
         {ADDRESS: address_receiver, AMOUNT: amount},
         {ADDRESS: address_giver, AMOUNT: change},
     ]
 
-    request = helpers.request.transfer(inputs, outputs, identifier=sdk_wallet[1])
+    request = helpers.request.transfer(inputs, outputs)
+    _, request._identifier = sdk_wallet
```

Comment:
`sdk_wallet` is a wallet instance, isn't it? Do we intentionally use it as `identifier`?

Reply:
It is not. It is sdk wallet handle with some did